web/src/features/charts/DataSources.tsx

```diff
@@ -55,12 +55,7 @@ export function DataSources({
             side="bottom"
             isMobile={isMobile}
           >
-            <div>
-              <IoInformationCircleOutline
-                className="text-emerald-800 dark:text-emerald-500"
-                size={20}
-              />
-            </div>
+            <Info className="text-emerald-800 dark:text-emerald-500" size={20} />
```

Comment:
I think the icon is a bit big here. I think either 16 or 18 is better

Reply:
I think 16 works but then we need to center align these otherwise it looks very strange, would that work?